system/include/emscripten/val.h

```diff
@@ -630,6 +640,12 @@ class val {
       : handle(handle), thread(pthread_self())
   {}
 
+  // Whether this value is a uses incref/decref (true) or is a special reserved
+  // value (false).
+  bool uses_refcount() const {
```

Comment:
nit: This file isn't really consistent but `uses_ref_count` matches a little better for `val` methods.